### ISSUE TITLE
[Accessibility]: Batches: No label for Batch Changes icon

### ISSUE THREAD

**Issue by BolajiOlajide** [sourcegraph/sourcegraph#34666]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34069

### Problem description

On the global batch changes list page, the batch change logo doesn't have a label so the screen reader reads it as `image`.

<img width="835" alt="Screenshot 2022-04-28 at 21 13 12" src="https://user-images.githubusercontent.com/25608335/165837696-75478399-0d1a-4e42-952b-f3554082ffea.png">
 

### Expected behavior

The screen reader should read the label of the icon.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
